scripts/exglobal_atmos_products.sh

```diff
@@ -42,7 +42,7 @@ fi
 # Section creating pressure grib2 interpolated products
 
 # Files needed by ${INTERP_ATMOS_MASTERSH}
-MASTER_FILE="${COM_ATMOS_MASTER}/${PREFIX}master.grb2${fhr3}"
+MASTER_FILE="${COM_ATMOS_MASTER}/${PREFIX}master${fhr3}.grb2"
```

Comment:
@WenMeng-NOAA 
The workflow looks for the master file to kick off the products generation job.
Please update [this](https://github.com/NOAA-EMC/global-workflow/blob/0237dba612a18c64eb513b6e186701cf37a1a5ef/workflow/rocoto/gfs_tasks.py#L1018) line to:
```diff
-                                   'history_file_tmpl': f'{self.cdump}.t@Hz.master.grb2#dep#'},
+                                   'history_file_tmpl': f'{self.cdump}.t@Hz.master#dep#.grb2'},
```

The analysis products job trigger also needs to be updated.  Please update [this](https://github.com/NOAA-EMC/global-workflow/blob/0237dba612a18c64eb513b6e186701cf37a1a5ef/workflow/rocoto/gfs_tasks.py#L908) line to:
```diff
-        data = f'{atm_master_path}/{self.cdump}.t@Hz.master.grb2anl'
+        data = f'{atm_master_path}/{self.cdump}.t@Hz.masteranl.grb2'
```

Finally, the offline UPP for analysis needs to write out the updated filename to `COM`.  Please update [these](https://github.com/NOAA-EMC/global-workflow/blob/0237dba612a18c64eb513b6e186701cf37a1a5ef/parm/post/upp.yaml#L26-L27) lines with:
```diff
-      - ["{{ DATA }}/GFSPRS.GrbF00", "{{ COM_ATMOS_MASTER }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.master.grb2anl"]
-      - ["{{ DATA }}/GFSPRS.GrbF00.idx", "{{ COM_ATMOS_MASTER }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.master.grb2ianl"]
+      - ["{{ DATA }}/GFSPRS.GrbF00", "{{ COM_ATMOS_MASTER }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.masteranl.grb2"]
+      - ["{{ DATA }}/GFSPRS.GrbF00.idx", "{{ COM_ATMOS_MASTER }}/{{ RUN }}.t{{ current_cycle | strftime('%H') }}z.masteranl.grb2.idx"]
```

Reply:
Note more updates will be needed in the `hpssarch_gen.sh` for these changes in analysis filenames.

Reply:
I will work on that.

Reply:
@WenMeng-NOAA 
I just looked at the EE2 implementation standards and the correct extension for grib2 files is `grib2`.  We should replace `grb2` with `grib2`.

Reply:
@aerorahul Just confirm with you that updates will  be in all master/flux/goes/pgrb2/pgrb2b.

Reply:
@aerorahul Updated.